backend/src/main/java/com/scottlogic/GMSv2/restcontroller/TeamController.java

```diff
@@ -29,6 +33,22 @@ public ResponseEntity<Team> getTeamById(@PathVariable(value="id") UUID id) {
             .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
   }
 
+  @GetMapping("/{id}/club")
```

Comment:
Hear me out... 😅 

I think I'd prefer this under the /clubs/ endpoint, e.g. `/clubs?teamId=...`, similar to how we do getTeams in the TeamsController. Not a big deal though, so happy to leave it like this if you prefer.

Reply:
I think I like it better as is 🤔 lets merge! we can discuss this later and refactor if we like